backend/src/test/java/woowacourse/touroot/travelogue/fixture/TravelogueTestFixture.java

```diff
@@ -0,0 +1,120 @@
+package woowacourse.touroot.travelogue.fixture;
+
+import java.util.List;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.stereotype.Component;
+import woowacourse.touroot.place.domain.Place;
+import woowacourse.touroot.travelogue.domain.Travelogue;
+import woowacourse.touroot.travelogue.domain.TravelogueDay;
+import woowacourse.touroot.travelogue.domain.TraveloguePhoto;
+import woowacourse.touroot.travelogue.domain.TraveloguePlace;
+import woowacourse.touroot.travelogue.dto.request.TravelogueDayRequest;
+import woowacourse.touroot.travelogue.dto.request.TravelogueLocationRequest;
+import woowacourse.touroot.travelogue.dto.request.TraveloguePhotoRequest;
+import woowacourse.touroot.travelogue.dto.request.TraveloguePlaceRequest;
+import woowacourse.touroot.travelogue.dto.request.TravelogueRequest;
+import woowacourse.touroot.travelogue.dto.response.TravelogueDayResponse;
+import woowacourse.touroot.travelogue.dto.response.TravelogueLocationResponse;
+import woowacourse.touroot.travelogue.dto.response.TraveloguePlaceResponse;
+import woowacourse.touroot.travelogue.dto.response.TravelogueResponse;
+
+@Component
+public class TravelogueTestFixture {
+
+    private TravelogueTestFixture() {
+    }
+
+    public static Travelogue getTravelogue() {
+        return new Travelogue("낭만의 시베리아 횡단철도 여행", "https://photo-key.jpeg");
+    }
```

Comment:
```suggestion
    public static final Travelogue SIVERIA_TRAVELOUGUE_ID_1 = new Travelogue("낭만의 시베리아 횡단철도 여행", "https://photo-key.jpeg");
    }
```
fixture는 매번 생성하는 메서드를 제공하는 역할이 아니라 미리 고정된 상수를 이용하는 것으로 알고 있습니다!
이렇게 하면 사용하는 쪽에서는 다음처럼 사용하게 될 거에요

```java

// given 
Travelogue travelogue = TravelogueTestFixture.SIVERIA_TRAVELOUGUE_ID_1
...
...
```
여러번 사용될 때 마다 생성 비용을 줄일 수 있는 것도 fixture의 장점이라고 생각합니다.
자꾸 픽스쳐 관련해서 말씀드려서 송구스러운데요 ㅎ.. 만약 시간 괜찮으시면 상수로 개선 부탁드립니다..!